pages/_app.tsx

```diff
@@ -21,7 +20,6 @@ export default class App extends NextApp {
       <ViewerProvider>
         <MDXProvider components={shortCodes}>
           <CacheProvider value={cache}>
-            {globalStyles}
             <AppLayout>
```

Comment:
I think that we will want to import Layout on page level, since that'll give us much more flexibility (e.g. to create a full width div with different background color), but that can happen later.